_data/finance.yml

```diff
@@ -25,8 +25,9 @@ websites:
     - name: FreeAgent
       url: https://www.freeagent.com
       img: freeagent.png
-      twitter: freeagent
-      tfa: No
+      tfa: Yes
+      software: Yes
+      docs: https://www.freeagent.com/support/kb/getting-started/two-step-verification
```

Comment:
@stephengroat in reply to your last PR comment - do you mean change this line away from https?  Thanks!